litestar/contrib/mongodb_motor/repository/_sync.py

```diff
@@ -0,0 +1 @@
+"""Waiting for feedback on async implementation before implementing sync version."""
```

Comment:
this can be autogenerated from the async code using the unasync pre-commit hook (ask @provinzkraut in discord on how to operate this).